README.md

```diff
@@ -48,6 +48,10 @@ cmake .. -D CMAKE_PREFIX_PATH=$CONDA_PREFIX -D CMAKE_INSTALL_PREFIX=$CONDA_PREFI
 make && make install
 ```
 
+Try Jupyter Lite demo by clicking below
+
+[![lite-badge](https://jupyterlite.rtfd.io/en/latest/_static/badge.svg)](https://github.com/jupyter-xeus/xeus-r/lab/index.html)
```

Comment:
This looks good, but maybe we should hold off on this until we integrate `untar.js`, otherwise it won't work 😅

Reply:
Yupp just raised it here :)